src/main/resources/hibernate.cfg.xml

```diff
@@ -0,0 +1,14 @@
+<hibernate-configuration>
+    <session-factory>
+        <property name="hibernate.dialect">org.hibernate.dialect.MariaDBDialect</property>
+        <property name="connection.url">jdbc:mysql://localhost:3306/ticket_app</property>
+        <property name="connection.username">root</property>
+        <property name="connection.password">003323119</property>
+        <property name="show_sql">true</property>
+        <property name="hbm2ddl.auto">update</property>
```

Comment:
We have learned about _create-drop_ property but maybe in this task its ok to use another one